lib/greenhouse_api/client.rb

```diff
@@ -33,18 +33,40 @@ def get_current_offer_for_application(application_id)
         http_method: :get,
         headers: headers,
         endpoint: "applications/#{application_id}/offers/current_offer",
-        params: {}
       )
+      create_response(response)
+    end
 
-      if response.status == 200
-        Response.new(
-          body: response.body,
-          headers: response.headers,
-          status: response.status
-        )
-      else
-        response
-      end
+    def create_user(on_behalf_of, body)
```

Comment:
Should we expect the `body` input to come in as json? Or should we call `.to_json` on the input?